gcc.symbols.imp

```diff
@@ -56,6 +56,8 @@
   { symbol: [ "imaxdiv_t", private, "<inttypes.h>", public ] },
   { symbol: [ "intmax_t", private, "<stdint.h>", public ] },
   { symbol: [ "intmax_t", private, "<inttypes.h>", public ] },
+  { symbol: [ "uintmax_t", private, "<stdint.h>", public ] },
```

Comment:
An argument could be made that `uintmax_t` should be sorted towards the end of this initialization list, but I think it makes sense to have it with the other `.*int.*` types.